doc/manuals/admin/build_source.md

```diff
@@ -85,16 +87,32 @@ for details. Recommended version is 3.6 (although 3.2 and 3.4 should also work f
 
 The Orion Context Broker comes with a suite of functional, valgrind and end-to-end tests that you can also run, following the following procedure (optional):
 
+* To install mongodb-org-shell using yum, create a /etc/yum.repos.d/mongodb.repo file.
+
+        [mongodb-org-3.6]
+        name=MongoDB Repository
+        baseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.6/x86_64/
+        gpgcheck=1
+        enabled=1
+        gpgkey=https://www.mongodb.org/static/pgp/server-3.6.asc
+
 * Install the required tools:
 
-        sudo yum install python python-flask pyOpenSSL curl nc mongodb-org-shell valgrind bc
+        sudo yum install python python-devel python-pip curl nc valgrind bc
```

Comment:
Looking to https://github.com/telefonicaid/fiware-orion/blob/master/ci/rpm7/build-dep.sh (which in some sense is "paralell" to the procedure described in build_source.md) I think `python-devel` is not needed here. 

Maybe `python-devel` is needed only for `libffi-devel`? In that case, "move" it to line 104 which should read as follows:

> In the case of the aarch64 architecture, additionally install libffi-devel and python-devel using yum. It is needed when building pyOpenSSL

Reply:
Your comment is correct. The python-devel is needed only for the libffi-devel. I edited line 100 and 104.

Reply:
Fixed in 6039fe1